csm_web/frontend/src/components/section/MentorSectionInfo.tsx

```diff
@@ -1,20 +1,37 @@
 import React, { useState } from "react";
-import { useSectionStudents } from "../../utils/queries/sections";
-import { Mentor, Spacetime, Student } from "../../utils/types";
+import { Navigate, Route, Routes } from "react-router-dom";
```

Comment:
There are a lot of ESLint warnings here about unused variables; you should delete the unused variables here.